ui/src/app/pages/signed-in/access-renewal-notification.tsx

```diff
@@ -46,9 +60,11 @@ export const AccessRenewalNotificationMaybe = (
     <NotificationBanner
       dataTestId='access-renewal-notification'
       text={`Time for ${accessType} renewal. ${timeLeft}`}
+      boxStyle={boxColor}
```

Comment:
optional: if these were named `boxStyle` and `iconStyle` we could reference them like
```
{...{boxStyle, iconStyle}}
```